pom.xml

```diff
@@ -277,15 +277,15 @@
                                 <mssql.image>mcr.microsoft.com/mssql/rhel/server:2022-latest</mssql.image>
                                 <!-- Oracle image - we aim to use same version as Dev Services for Oracle so that we only download one image -->
                                 <oracle.image>docker.io/gvenzl/oracle-free:23-slim-faststart</oracle.image>
-                                <db2.image>quay.io/quarkusqeteam/db2:11.5.7.0</db2.image>
+                                <db2.image>docker.io/ibmcom/db2:11.5.8.0</db2.image>
                                 <mongodb.image>docker.io/library/mongo:5.0</mongodb.image>
                                 <redis.image>docker.io/library/redis:7.2</redis.image>
                                 <wiremock.image>docker.io/wiremock/wiremock:3.3.1</wiremock.image>
                                 <!-- TODO use official image when/if this fixed https://github.com/hashicorp/docker-consul/issues/184 -->
                                 <consul.image>docker.io/bitnami/consul:1.17.2</consul.image>
                                 <infinispan.image>quay.io/infinispan/server:14.0</infinispan.image>
                                 <infinispan.expected-log>Infinispan Server.*started in</infinispan.expected-log>
-                                <spring.cloud.server.image>quay.io/quarkusqeteam/spring-cloud-config-server:3.0</spring.cloud.server.image>
+                                <spring.cloud.server.image>docker.io/hyness/spring-cloud-config-server:3.1-jdk17</spring.cloud.server.image>
```

Comment:
This is not official image. I appreciate that https://github.com/hyness/spring-cloud-config-server offers also the newest Spring Cloud Config versions, but the project doesn't really build a confidence in me that we should run it's code without inspecting it line by  line inside of our VPN and our Jenkins jobs.

- https://github.com/hyness/spring-cloud-config-server/blob/main/src/main/kotlin/org/freshlegacycode/cloud/config/server/ConfigServerApplication.kt looks fine, but were is a guarantee that someone will not merge there arbitrary code and we don't automatically bump this image without inspecting the repo code?
- We don't really have a guarantee that what is in the GitHub repo is exactly what the Docker image uses. How do we know what https://github.com/hyness put there?
- Where is even guarantee that https://hub.docker.com/r/hyness/spring-cloud-config-server/tags is uploaded by the same person?

Reply:
It's not that I disagree with you.
What I know about this is that this image repository was used but was changed here https://github.com/quarkus-qe/quarkus-test-suite/pull/392/files

Also we should probably replace the link to it in readme here https://github.com/quarkus-qe/quarkus-test-suite/tree/main/spring/spring-cloud-config#quarkus-spring-cloud-config

I'll will change it back to our image, just we probably should be aware that we testing it with image older more then 2 years.

Reply:
> What I know about this is that this image repository was used but was changed here https://github.com/quarkus-qe/quarkus-test-suite/pull/392/files

Yes that makes your change absolutely understandable, but that was someone else decision. It doesn't make it right.

Reply:
> Also we should probably replace the link to it in readme here https://github.com/quarkus-qe/quarkus-test-suite/tree/main/spring/spring-cloud-config#quarkus-spring-cloud-config

+1

> I'll will change it back to our image, just we probably should be aware that we testing it with image older more then 2 years.

let's do something about it, discuss it in the team meeting

Reply:
meeting is not necessary, let's just create JIRA or something